Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AdAuthManager.java

```diff
@@ -180,8 +182,22 @@ public AuthResult signIn(@Nullable AuthResult savedAuth) throws IOException {
                 //TODO: should narrow to AuthError.InteractionRequired
                 ac1.acquireToken(env.managementEndpoint(), true, userId, isDisplayable);
             }
-            ac1.acquireToken(env.resourceManagerEndpoint(), false, userId, isDisplayable);
+
+            // FIXME!!! Some environments and subscriptions can't get the resource manager token
+            // Let the log in process passed, and throwing the errors when to access those resources
+            try {
+                ac1.acquireToken(env.resourceManagerEndpoint(), false, userId, isDisplayable);
+            } catch (AuthException ignored) {
+            }
```

Comment:
CommonSettings.getEnvironment() instanceOf ProvidedEnvironment? Otherwise still throw exception?

Reply:
NP, fix it!